cmd/kube-controller-manager/app/options/options.go

```diff
@@ -104,6 +104,7 @@ func NewCMServer() *CMServer {
 			ClusterSigningCertFile:   "/etc/kubernetes/ca/ca.pem",
 			ClusterSigningKeyFile:    "/etc/kubernetes/ca/ca.key",
 			ReconcilerSyncLoopPeriod: metav1.Duration{Duration: 5 * time.Second},
+			EnableTaintManager:       true,
```

Comment:
This was false in a previous commit iirc, and we decided it has to be disabled this release. 

/cc @davidopp